pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -314,6 +314,11 @@ sudoPrompt:
   explanation: 'This will modify the following paths:'
   buttonText: OK
 
+Required:
```

Comment:
This name could be more descriptive. Maybe something like `unsupportedEnvironment`? `unmetRequirements`?

Reply:
`unmetPrerequisites`?

Reply:
changed